README.md

```diff
@@ -25,11 +25,25 @@ Import the 3box module
 ```js
 const Box = require('3box')
 ```
-or use the dist build in your html code
+Import using the dist build in your html code
 ```js
 <script type="text/javascript" src="../dist/3box.js"></script>
 ```
 
+Or optionally by loading remote copy from [unpkg](https://unpkg.com/) CDN.
+
+```html
+<!-- The most recent version  -->
+<script src="https://unpkg.com/3box/dist/3box.js"></script>
+<!-- The most recent minified version  -->
+<script src="https://unpkg.com/3box/dist/3box.min.js"></script>
+<!-- Load specific versions by specifying the version as follows -->
+<script src="https://unpkg.com/3box@<version>/dist/3box.js"></script>
+```
+
+You can also load it using a <script> using the unpkg CDN or the jsDelivr CDN. Inserting one of the following lines will make a Ipfs object available in the global namespace.
```

Comment:
You mean the Box object and not the Ipfs object right? 😂

Reply:
ahh yeah, don't want any of that line :) pushed up a fix